src/pages/Lending/hooks/useGetEstimatedFeesQuery.ts

```diff
@@ -5,21 +5,38 @@ import type { EstimateFeesInput } from 'components/Modals/Send/utils'
 import { estimateFees } from 'components/Modals/Send/utils'
 import { bn } from 'lib/bignumber/bignumber'
 import { fromBaseUnit } from 'lib/math'
-import { selectAssetById, selectMarketDataByAssetIdUserCurrency } from 'state/slices/selectors'
-import { useAppSelector } from 'state/store'
+import {
+  selectAssetById,
+  selectFeeAssetById,
+  selectMarketDataByAssetIdUserCurrency,
+} from 'state/slices/selectors'
+import { store, useAppSelector } from 'state/store'
 
 // For use outside of react with queryClient.fetchQuery()
 export const queryFn = async ({ queryKey }: { queryKey: EstimatedFeesQueryKey }) => {
   const { estimateFeesInput, asset, assetMarketData } = queryKey[1]
 
   // These should not be undefined when used with react-query, but may be when used outside of it since there's no "enabled" option
-  if (!asset || !estimateFeesInput?.to || !estimateFeesInput.accountId) return
+  if (
+    !asset ||
+    !estimateFeesInput?.to ||
+    !(estimateFeesInput.accountId || estimateFeesInput.pubkey)
+  )
+    return
+
+  const feeAsset = selectFeeAssetById(store.getState(), asset.assetId)
```

Comment:
This was 100% wrong previously - fees are in native asset and we would get them wrong by orders of magnitude for tokens